frontend/src/components/pages/friends/Friends.tsx

```diff
@@ -15,7 +15,7 @@ const Friends: React.FC = () => {
 
   useEffect(() => {
     getFriends(id).then(users => setFriends(users));
-  }, [id]);
+  }, []);
```

Comment:
This will cause eslint error on production build

Reply:
Yes, I saw it, that I have some different rows with develop, because I added label "don't merge"